docs/en/sql-reference/statements/create/table.md

```diff
@@ -112,6 +112,13 @@ Materialized expression. Such a column can’t be specified for INSERT, because
 For an INSERT without a list of columns, these columns are not considered.
 In addition, this column is not substituted when using an asterisk in a SELECT query. This is to preserve the invariant that the dump obtained using `SELECT *` can be inserted back into the table using INSERT without specifying the list of columns.
 
+### EPHEMERAL {#ephemeral}
+
+`EPHEMERAL expr`
+
+Ephemeral column. Such a column isn't stored in the table and cannot be SELECTed, but can be referenced in the defaults of CREATE statement.
+Currently SELECT/INSERT invariant is not preserved for this type, i.e. set of fields obtained using `SELECT *` cannot be inserted back into the table using INSERT without specifying the list of columns.
```

Comment:
inconvenient, maybe it's better to do the opposite - so that the insertion into this EPHEMERAL requires an explicit listing of this column in the INSERT statement?

I.e.
```
INSERT INTO table VALUES <-- nothing is inserted into the ephemeral column
INSERT INTO table(col1,col2,ephemeral) VALUES <-- value inserted into ephemeral column 
```

Reply:
Yes, I think it will be a good solution - after all ephemerals were proposed to fill defaults which are referencing ephemerals, but if all fields are inserted ephemerals are not needed in such insert.

Reply:
Agree, but default ephemeral columns should not appear neither in `SELECT *` for in `INSERT`.